packages/sdk/src/types/Orders.ts

```diff
@@ -92,3 +92,118 @@ export interface OrdersListResponse {
 	data_ref: 'orders'
 	orders: OrdersGetResponse[]
 }
+
+//domain types
+export type OrderRequest = {
```

Comment:
I don't like the idea of naming the domain type with postfix request, response etc.
BUT at the same time, I have no idea how to create it differently :) 
At first I would probably create a type: `QualificationsCheckEligibilityOrderResponse/Request` but I understand that `OrderRequest` and `OrderResponse` are different and can be used elsewhere in the future, so creating just the `Order` type is impossible here 👍🏼

Reply:
As we have learned while working on openAPI, this patern must be used.
Domain types very often differ if object is used in requests or responses. Until someone will give a better solution, it must be like that.

Reply:
what?

Reply:
Modele często różnią się w zależności czy są używane w zapytaniach czy odpowiedziach (czytaj. pola są różne).
Ozacza to, że jeżeli chcemy tworzyć `typy domenowe` musimy tworzyć typy domenowe różne zależnie od tego czy używamy ich w zapytaniach czy odpowiedziach.
Taki system przyjeliśmy tworząc OpenAPI.